samples/subsys/zbus/uart_bridge/prj_sparkfun_red_v_things_plus.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_LOG=y
+CONFIG_LOG_MODE_MINIMAL=y
+CONFIG_THREAD_NAME=y
+
+CONFIG_SERIAL=y
+CONFIG_UART_LINE_CTRL=n
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_SIFIVE_PORT_1=y
+CONFIG_UART_SIFIVE_PORT_1_TXCNT_IRQ=1
+CONFIG_UART_SIFIVE_PORT_1_RXCNT_IRQ=1
+
+CONFIG_ZBUS=y
```

Comment:
This is a surprisingly specific test to add support to... more of a curiosity than anything else, why adding support for this specific one?

Reply:
The direct answer is to pass the CI tests.
The twister runs this test because UART1 is enabled.

https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/subsys/zbus/uart_bridge/sample.yaml#L17-L20
```
    filter: dt_nodelabel_enabled("uart1")
```

Internally reason:
The PR is mostly copied from hifive1_revb because the board configuration is very similar.
The .prj file also so.
Compared to hifive_revb with this board, The only difference is it doesn't have WiFi and some LED and has less Flash.
(As you know, SparkFun's document uses this board as hifive1_revb...)

We can select the option that disables UART1.
But to make it easy to manage the configuration, I think it is better to make it similar to the hifive1_revb configurations.

Reply:
> Can you reduce the resolution of the board image? Should aim to be <50kB, all pictures have been recently converted to be <750px on the major dimension and 75% jpeg quality.

Resized and reduced the image.

> Thanks for adding support for this! I happen to own one of these boards and always used a hacked config with it :-)
👍